### PR TITLE
add jquery.lazy v0.5.4 with git autoupdate

### PR DESCRIPTION
**Pull request by maruilian11**

### CODE REVIEW

File: ajax/libs/jquery.lazy/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "node": "*"
+    },
+    "dependencies":
+    {
+        "jquery": ">=1.7.0"
+    },
+    "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/eisbehr-/jquery.lazy.git",
+    "basePath": "./",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.lazy/package.json]: No need to set project's root as basePath.

File: ajax/libs/jquery.lazy/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    "files": [
+    "jquery.lazy.min.js",
+    "jquery.lazy.js"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.lazy/package.json]: Wrong indent again, as #4773

**Comment by maruilian11** [comment]
problems fixed, please take a look, thx~

**Comment by PeterDaveHello** [comment]
https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#d-adding-a-new-library
> If you will create its package.json, the indent of it must be 2 spaces, others will be fine to follow official version or to use 2 spaces.

**Pull request merged**